src/jit/lclvars.cpp

```diff
@@ -6491,6 +6601,14 @@ void Compiler::lvaDumpEntry(unsigned lclNum, FrameLayoutState curState, size_t r
     {
         printf(" stack-byref");
     }
+    if (varDsc->lvClassHnd != nullptr)
+    {
+        printf(" class-hnd");
```

Comment:
> I could add this as a dump option, though we seem to have perhaps too many dump options already

My inclination would be not to add a dump option, and leave how you have it -- thanks for explaining.